gui/src/components/CopyJobSpec.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography'
+import { withStyles } from '@material-ui/core/styles';
+import { CopyToClipboard } from "react-copy-to-clipboard";
+
+const styles = theme => ({
+  button: {
+    margin: theme.spacing.unit,
+  },
+  inform: {
+	display: "inline-block",
+    opacity: "1",
+  }
+});
+    
+
+class Copy extends React.Component {
+  state = {
+    copied: false
+  };
+
+  render() {
+    return (
+      <div>
+        <CopyToClipboard text={this.props.JobSpec} onCopy={() => this.setState({ copied: true })}>
```

Comment:
How do you feel about making this component generic so that it can be used in other contexts? i.e. it could be in the file `Copy.js` and take a generic `data` + `buttonText` prop.